api/v1alpha4/machinehealthcheck_types.go

```diff
@@ -45,6 +45,12 @@ type MachineHealthCheckSpec struct {
 	// +optional
 	MaxUnhealthy *intstr.IntOrString `json:"maxUnhealthy,omitempty"`
 
+	// Any further remediation is only allowed if the number of machines selected by "selector" as not healthy
+	// is within the range of "UnhealthyRange". Takes precedence over MaxUnhealthy.
+	// +optional
+	// +kubebuilder:validation:Pattern=^\[[0-9]+-[0-9]+\]$
+	UnhealthyRange *string `json:"unhealthyRange,omitempty"`
```

Comment:
Could we provide an example on how to use the range in the comment?

Reply:
Looking at the code below, is this range common in Kubernetes API? Do we have an example on how it's being used and implemented in other APIs?

Reply:
The closet I could find in kubernetes was `limit ranges`. for cpu, but I'm not sure if that's what we really want. Another one was `service-node-port-range` in api range, but again that was very specific to ports.